assets/script/script.js

```diff
@@ -31,6 +31,15 @@ $(document).ready(function () {
       }
     }
   });
+
+  //Listener on clear button clears the history list
+  $(".shiftRight").on("click", function () {
+    window.localStorage.clear("history");
+    window.sessionStorage.clear();
```

Comment:
Also, are we using sessionStorage anywhere? I think we're just using localStorage to save the history right? So, we dont want to clear the sessionStorage too if we're not using it.

Reply:
No, the sessionStorage is not being used, i forgot to erase it.